pkg/cable/libreswan/libreswan.go

```diff
@@ -175,7 +176,7 @@ func retrieveActiveConnectionStats() (map[string]int, map[string]int, error) {
 	defer cancel()
 
 	// Retrieve active tunnels from the daemon
-	cmd := exec.CommandContext(ctx, "/usr/libexec/ipsec/whack", "--trafficstatus")
+	cmd := exec.CommandContext(ctx, "/usr/sbin/ipsec", "whack", "--trafficstatus")
```

Comment:
this is marginally better, but the best is to use "ipsec" without a path, so it works as long as it is in the path (/sbin or /usr/sbin or /usr/local/sbin or whatever compile time options are used). Using "ipsec <cmd> <options>" is supported specifically to avoid any hardcoded paths. if exec.CommandContext uses regular PATH handling, this is best run without the hardcoded path

Reply:
Yeah, I can modify the PR to use `exec.LookPath("ipsec")`, maybe somewhere in init.